ext/dom/tests/gh11906.phpt

```diff
@@ -0,0 +1,37 @@
+--TEST--
+GH-11906 (prepend without children after creating fragment results in segfault)
+--EXTENSIONS--
+dom
+--FILE--
+<?php
+$doc = new DOMDocument;
+$doc->loadXML(<<<XML
+<?xml version="1.0"?>
+<container>
+    <child/>
+</container>
+XML);
+
+$container = $doc->documentElement;
+$child = $container->firstElementChild;
+
+$test = $doc->createElement('foo');
+$test->append($child);
+echo $doc->saveXML();
+echo $doc->saveXML($test), "\n";
+$test->prepend($child);
+echo $doc->saveXML();
+echo $doc->saveXML($test), "\n";
+$test->append($child);
```

Comment:
I can't make out where the output of one ``echo`` ends and where another ``echo`` starts.

Maybe add some ``echo "----------\n";` to make it more obvious?

Reply:
I've done so in the merge. If you're interested: https://github.com/php/php-src/commit/d19e4da125365080b52c4e6e4a1b2e105c14add5#diff-8386e35f446a7f90b1425135e71fddb65f86c1c115255c0a454878881e7beeb2